src/utils.js

```diff
@@ -264,9 +264,10 @@ export const getPlotContentFromItemId = itemId => ({
 
 /**
  * @param {farmhand.plotContent} plotContent
- * @returns {string}
+ * @returns ?{string}
```

Comment:
The `?` needs to go inside of the braces here, I think. So, `{?string}`

Reply:
oops!